backend/lucene/src/main/java/org/hibernate/search/backend/lucene/impl/LuceneBackendFactory.java

```diff
@@ -106,21 +123,21 @@ private Version getLuceneVersion(EventContext backendContext, ConfigurationPrope
 		else {
 			Version latestVersion = LuceneBackendSettings.Defaults.LUCENE_VERSION;
 			log.recommendConfiguringLuceneVersion(
-					LUCENE_VERSION.resolveOrRaw( propertySource ),
-					latestVersion,
-					backendContext
+				LUCENE_VERSION.resolveOrRaw( propertySource ),
+				latestVersion,
+				backendContext
 			);
 			luceneVersion = latestVersion;
 		}
 		return luceneVersion;
 	}
 
 	private BeanHolder<? extends DirectoryProvider> getDirectoryProvider(EventContext backendContext,
-			BackendBuildContext buildContext, ConfigurationPropertySource propertySource) {
```

Comment:
Please remove all these format changes from your PR.

Reply:
I tried to find parameters for it. But unfortunately this format is just Net Baeans. It has quite flexible parameters. I don't know how to InterliJ, elae eclise has several incompatible rules. I have been programming for NB for 17 years, earlier several years in Borland environments. I programmed on Eclipse, but someone is not possible, it is too small and not readable for my needs. It's difficult to change to Eclippse in its entirety, and switching is difficult as you implement other projects. I was looking for a solution for this indentation, but not everything can be done identically as in Eclipse, it sometimes shifts automatically as it formats the source. I remove these indentations by hand. But it doesn't always work out and it costs a lot of time, and the differences can only be seen after you commit.

Reply:
I just had a look at Netbeans 11.3's formatting configuration, and from what I can see it is the *default* to add two tabs when a class/method declaration spreads over multiple lines. You should check that in the "Alignment" category, all checkboxes under "Multiline Alignment" are disabled.

Also, to avoid that kind of problem, you can select the text your want to format before formatting it with "ALT+SHIFT+F", instead of formatting the whole file. That way you'll avoid formatting pre-existing, unrelated code.